src/components/project/project-member/project-member.repository.ts

```diff
@@ -31,7 +31,7 @@ export class ProjectMemberRepository extends DtoRepository<
     super();
   }
 
-  async verifyRelationshipEligibility(projectId: ID, userId: ID) {
+  async verifyRelationshipEligibility?(projectId: ID, userId: ID) {
```

Comment:
I think you can remove the optional operator `?` and adjust the service file per my comment below.